main.go

```diff
@@ -140,11 +144,22 @@ func main() {
 	globalHTTPTimeout := time.Duration(globalHTTPTimeoutMS) * time.Millisecond
 	eventRecorder := mgr.GetEventRecorderFor("keda-operator")
 
+	kubeClientset, _ := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
```

Comment:
can't we get the client from `mgr.GetClient()`?

Reply:
Same as above